api/swagger.yaml

```diff
@@ -3775,11 +3811,20 @@ components:
           type: string
           enum:
             - boxtruck
-            - tractor
+            - truck
             - trailer
+            - lorry
+            - tractor
             - airplane
             - deep sea vessel
             - barge
+            - passenger_car
+            - motor_bike
```

Comment:
This one has an underscore, while `deep sea vessel` has spaces. What is the recommended format for these enums?

Reply:
Entity types are formatted using camelCase, maybe we could introduce more uniformity by giving every enum the same format?